chasm/lib/scheduler/backfiller_tasks.go

```diff
@@ -61,17 +59,15 @@ func (b *BackfillerTaskExecutor) Execute(
 
 	scheduler, err := backfiller.Scheduler.Get(ctx)
 	if err != nil {
-		return fmt.Errorf("%w: %w",
-			serviceerror.NewInternal("scheduler tree missing node"),
-			err)
+		b.baseLogger.Error("scheduler tree missing node", tag.Error(err))
+		return ErrUnprocessable
```

Comment:
You can use `NewUnprocessableTaskError` from `service/history/queues` here and include all of the error details.